src/bci_build/package/python/README.md.j2

```diff
@@ -0,0 +1,56 @@
+# {{ image.pretty_name }} container image
+
+{% include 'badges.j2' %}
+
+## Description
+
+[Python](https://www.python.org/) is an interpreted, interactive, object-oriented, open-source programming language. It incorporates modules, exceptions, dynamic typing, high-level dynamic data types, and classes. It provides interfaces to many system calls, libraries, and various window systems, and it is extensible in C or C++. It is also usable as an extension language for applications that require programmable interfaces.
+
+## Usage
+
+To deploy an application, install dependencies, copy the sources, and configure the application's main script:
+
+```Dockerfile
+FROM {{ image.pretty_reference }}
+
+WORKDIR /app
+
+COPY requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+CMD [ "python3", "./main-script.py" ]
+```
+
+Build and run the container image:
+
+```ShellSession
+$ podman build -t my-python-app .
+$ podman run -it --rm my-python-app
+```
+
+To run a single script inside a container, use the following command:
+
+```ShellSession
+$ podman run --rm -v "$PWD":/app:Z -w /app {{ image.pretty_reference }} python3 script.py
+```
+
+## Additional tools
+
+The following tools are included in the image:
+
+{% set version = image.version|replace(".", "") -%}
+
+- python{{ version }}-devel
+- python{{ version }}-pip
+{% if image.has_pipx -%}
+- python{{ version }}-pipx
+{% endif -%}
+{% if image.has_wheel -%}
+- python{{ version }}-wheel
```

Comment:
is it worth listing the `python3-devel` package here? otherwise I need to add a check `if is_python_system` because it will be `python3-devel` for the system version, not for 3.11 on SLE.

Reply:
It is not worth listing `python3-devel` imho, but the package names are incorrect for pip & wheel too. It's `python3-pip` for python 3.6 and not `python36-pip`

Reply:
I reworked this section, no version number in names anymore.